clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1380,10 +1379,31 @@ Sample usage:
   // Setting it as a C++11 attribute is also valid in a C++ program.
   // void convfunc(void) [[clang::convergent]];
 
-  int f() {
-    [[clang::convergent]] foo(arg);
-    // The call to 'foo' has attribute 'convergent'.
+  }];
+}
+
+def NoConvergentDocs : Documentation {
+  let Category = DocCatFunction;
+  let Content = [{
+The ``noconvergent`` attribute removes the LLVM ``convergent`` attribute if
+present. If a statement is marked ``noconvergent`` and contains calls,
+``convergent`` attributes on those calls are removed as well.
```

Comment:
This is describing an implementation detail, which doesn't belong in the user facing documentation